sphinx_gallery/gen_rst.py

```diff
@@ -163,7 +163,7 @@ def __exit__(self, type_, value, tb):
 """
 
 # Header used to include raw html
-html_header = """.. only:: builder_html
+html_header = """.. only:: builder_html or readthedocs
```

Comment:
I'm not sure if we even need this line given that it's wrapped in a `.. raw:: html`. It seems redundant, and thus simpler just to remove this `.. only::` line altogether